packages/gestalt/src/Masonry.js

```diff
@@ -145,7 +142,24 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
     this.setState({
       scrollTop: getScrollPos(scrollContainer),
     });
-  });
+  };
+
+  scrollTimeoutFn: ?TimeoutID = null;
```

Comment:
nit: this isn't a function, it's a timeout id, so the name seems a bit misleading